### PR TITLE
Generate more dummy data via an online generator, add some info on it…

### PR DESCRIPTION
**Pull request by gashcrumb**
… too and the code used to generate the data

### CODE REVIEW

**Comment by gashcrumb** [comment]
@KurtStam looks like to have stuff like `organizationId` and `componentId` in normal camel case I'll have to do a bit of digging, first pass of trying that broke the associations.

File: docs/dataGenerator.txt [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  {
+    'repeat(10)': {
+      model: 'User',
+      data: {
+        firstName: '{{firstName()}}',
+        lastName: '{{surname()}}',
+        username: function() {
+          return this.firstName.toLowerCase().charAt(0) + this.lastName.toLowerCase(); 
+        },
+        RoleId: '{{integer(1, 3)}}',
```
Comment (jimmidyson) [inline docs/dataGenerator.txt]: Guess this should be starting with lowercase letter? And OrganizationId too.

File: docs/dataGenerator.txt [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    }
+  },
+  {
+    'repeat(10)': {
+      model: 'IntegrationTemplate',
+      data: {
+        id: '{{index() + 1}}',
+        name: '{{lorem(1, "words")}}',
+        OrganizationId: 1,
+        UserId: '{{integer(1, 10)}}',
```
Comment (jimmidyson) [inline docs/dataGenerator.txt]: Looks like anything that's been updated to use Id suffix actually.

Comment (gashcrumb) [inline docs/dataGenerator.txt]: yeah, as I mentioned so far it appears that's gotta match the model name, am still looking at it.

**Comment by gashcrumb** [comment]
Hmm, seems if I try and rename the column used for the attribute so that the data gets loaded I can't get the API to return the assiciated object, for example this kind of URL would return the connection + the organization, user and component as well:

`http://localhost:9090/v1/connections/1?include=user&include=component&include=organization`

as an example using the code on this branch currently:

```
{
    "id": 1,
    "name": "incididunt",
    "description": "labore ut proident et labore ipsum Lorem eu amet est",
    "icon": "fa-linkedin-square",
    "configuredProperties": "{\"esse\":{\"name\":\"esse\",\"title\":\"esse\",\"description\":\"ad culpa excepteur esse ex\",\"type\":\"string\",\"enabled\":true,\"required\":true},\"excepteur\":{\"name\":\"excepteur\",\"title\":\"excepteur\",\"description\":\"cillum cillum eu dolor consequat\",\"type\":\"password\",\"enabled\":true,\"required\":true},\"proident\":{\"name\":\"proident\",\"title\":\"proident\",\"description\":\"ullamco mollit aliqua adipisicing labore\",\"type\":\"password\",\"enabled\":true,\"required\":true},\"qui\":{\"name\":\"qui\",\"title\":\"qui\",\"description\":\"ea anim labore velit ea\",\"type\":\"boolean\",\"enabled\":true,\"required\":false},\"mollit\":{\"name\":\"mollit\",\"title\":\"mollit\",\"description\":\"in magna mollit irure aliqua\",\"type\":\"boolean\",\"enabled\":true,\"required\":true},\"cillum\":{\"name\":\"cillum\",\"title\":\"cillum\",\"description\":\"nostrud eiusmod labore commodo officia\",\"type\":\"boolean\",\"enabled\":true,\"required\":true},\"sit\":{\"name\":\"sit\",\"title\":\"sit\",\"description\":\"irure consectetur enim dolore sint\",\"type\":\"password\",\"enabled\":true,\"required\":true},\"laborum\":{\"name\":\"laborum\",\"title\":\"laborum\",\"description\":\"consequat exercitation id ea exercitation\",\"type\":\"number\",\"enabled\":true,\"required\":false},\"et\":{\"name\":\"et\",\"title\":\"et\",\"description\":\"ut pariatur aliqua consectetur incididunt\",\"type\":\"boolean\",\"enabled\":true,\"required\":false}}",
    "position": "Anywhere",
    "verified": false,
    "createdAt": "2016-12-07T20:58:52.350Z",
    "updatedAt": "2016-12-07T20:58:52.350Z",
    "ComponentId": 24,
    "OrganizationId": 1,
    "UserId": 1,
    "User": {
        "id": 1,
        "username": "tgriffith",
        "uuid": null,
        "type": null,
        "email": null,
        "firstName": "Tucker",
        "hash": null,
        "lastName": "Griffith",
        "password": null,
        "status": null,
        "createdAt": "2016-12-07T20:58:41.271Z",
        "updatedAt": "2016-12-07T20:58:41.271Z"
    },
    "Organization": {
        "id": 1,
        "name": "Acme",
        "createdAt": "2016-12-07T20:58:40.745Z",
        "updatedAt": "2016-12-07T20:58:40.745Z"
    },
    "Component": {
        "id": 24,
        "name": "reprehenderit",
        "description": "enim ad veniam dolore velit exercitation occaecat in duis magna",
        "icon": "fa-twitter",
        "properties": "{\"culpa\":{\"name\":\"culpa\",\"title\":\"culpa\",\"description\":\"officia reprehenderit incididunt deserunt et\",\"type\":\"number\",\"enabled\":true,\"required\":true},\"adipisicing\":{\"name\":\"adipisicing\",\"title\":\"adipisicing\",\"description\":\"nostrud anim exercitation qui ullamco\",\"type\":\"boolean\",\"enabled\":true,\"required\":false},\"in\":{\"name\":\"in\",\"title\":\"in\",\"description\":\"enim consequat sint mollit officia\",\"type\":\"password\",\"enabled\":true,\"required\":true},\"cillum\":{\"name\":\"cillum\",\"title\":\"cillum\",\"description\":\"tempor fugiat ipsum velit dolore\",\"type\":\"boolean\",\"enabled\":true,\"required\":false},\"voluptate\":{\"name\":\"voluptate\",\"title\":\"voluptate\",\"description\":\"do eu aute culpa aute\",\"type\":\"boolean\",\"enabled\":true,\"required\":false}}",
        "remaining": null,
        "uri": null,
        "createdAt": "2016-12-07T20:58:49.076Z",
        "updatedAt": "2016-12-07T20:58:49.076Z",
        "ComponentGroupId": 21
    }
}
```

If I try and change the sequelize code to user organizationId for the column I can get the ID from the test data to go in there, but I can't get the above response back, and that's totally :-(

@kahboom thoughts?

**Comment by gashcrumb** [comment]
Gonna merge this so the data is in place at least, will continue looking at the attribute names in a separate PR.

**Comment by kahboom** [comment]
@gashcrumb - PMing you on IRC now

**Pull request merged**